### PR TITLE
Disable Guest Login Temporarily 

### PR DESCRIPTION
**Pull request by MPierre9**
Addresses #64 

Disables guest login so we can develop Google Auth first

### CODE REVIEW

File: browser_extension/js/main.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ $(function () {
   var $loginPage = $('.login.page'); // The login page
   var $chatPage = $('.chat.page'); // The chatroom page
   var $googleSignInButton = $('#btnGoogleSignIn'); // Google login button
-  var $guestSignInButton = $('#btnGuestLogin'); //Guest login button
+  // var $guestSignInButton = $('#btnGuestLogin'); //Guest login button - disabled
```
Comment (marcobeltempo) [inline browser_extension/js/main.js]: Delete

File: browser_extension/js/main.js [hunk 2]
```diff
@@ -291,9 +291,9 @@ $(function () {
   });
 
   // Reveals the guest username input field
-  $guestSignInButton.click(function () {
+  /* $guestSignInButton.click(function () {
```
Comment (marcobeltempo) [inline browser_extension/js/main.js]: Delete

File: browser_extension/src/browser_action/browser_action.html [hunk 3]
```diff
@@ -42,9 +42,9 @@
           <div class="row">
             <div class="col-md-10 .offset-md-2">
               <div class="panel-body" id="guestSignInBody">
-                <input type="button" id="btnGuestLogin" class="btn btn-basic btn-block" value="Guest Login" />
-                <input class="form-control usernameInput" id="guestUsername" placeholder="Username" type="text" style="display:none" maxlength="14"
-                />
+                <!--<input type="button" id="btnGuestLogin" class="btn btn-basic btn-block" value="Guest Login" />-->
```
Comment (marcobeltempo) [inline browser_extension/src/browser_action/browser_action.html]: Delete

**Comment by MPierre9** [comment]
@marcobeltempo should be merged with dev or chaddon v0.2.0?

**Pull request merged**